ome-demoserver.yml

```diff
@@ -47,15 +47,6 @@
         - python-markdown
         - mencoder # For the 'make movie' script
 
-    # Set desired state of webapps (upgrade v.s. install/keep current version)
-    - name: OMERO.web plugin upgrade | set default state "installed"
-      set_fact:
-        webapps_state: present
-    - name: OMERO.web plugin upgrade | if --extra-vars upgrade_webapps == True, allow web plugin(s) to upgrade.
-      set_fact:
-        webapps_state: latest
-      when: upgrade_webapps is defined and upgrade_webapps == "True"
```

Comment:
I'm fine with this simplification, but I'm unsure who might currently be depending on this. @manics, do you know?

Reply:
I think it's old stuff that never got cleaned up